package.json

```diff
@@ -23,7 +23,12 @@
     "all:clean": "rm -rf ./packages/**/dist && rm -rf ./apps/**/dist && rm -rf ./packages/**/tsconfig.build.tsbuildinfo",
     "all:build": "lerna run build",
     "all:version": "lerna version $1 --yes --no-push --no-git-tag-version --exact",
-    "all:publish": "lerna exec --no-private -- npm publish --tag latest --access public"
+    "all:publish": "lerna exec --no-private -- npm publish --tag latest --access public",
+    "apps:ocean": "nest start ocean-api"
```

Comment:
should not have start function within root package, if we need to we can use `npm run start -workspace=ocean-api`.